src/test/java/org/mdbenefits/app/journeys/MdBenefitsFlowJourneyTest.java

```diff
@@ -613,13 +610,23 @@ void completeFlow() {
         testPage.selectRadio("isApplicantApplying", "true");
         testPage.clickContinue();
 
-        //sex placeholder
-        assertThat(testPage.getTitle()).isEqualTo("Scaffold");
+        assertThat(testPage.getTitle()).isEqualTo(message("personal-info.sex.title"));
+        testPage.selectRadio("applicantSex", "M");
         testPage.clickContinue();
 
         assertThat(testPage.getTitle()).isEqualTo(message("applicant-school-enrollment.title"));
         testPage.selectRadio("applicantIsEnrolledInSchool", "Yes");
 
```

Comment:
I feel that conditional page behavior is best tested in smaller tests rather than the `completeFlow`. Let's discuss.